### PR TITLE
fix created va and updated va

### PR DESCRIPTION
**Pull request by ayusrina**

### CODE REVIEW

**Comment by sayaandreas** [comment]
conflict jah

File: api/virtual_accounts/virtual_accounts.go [hunk 1]
```diff
@@ -178,11 +236,29 @@ func (va *VAHandler) Create(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	fmt.Fprintf(w, "VA Number: %v Created!\n", vam.VaNum)
+	response := VAResponse{
+		Status:       1,
```
Comment (celynolivia) [inline api/virtual_accounts/virtual_accounts.go:279]: status change to "success"

File: api/virtual_accounts/virtual_accounts.go [hunk 2]
```diff
@@ -178,11 +236,29 @@ func (va *VAHandler) Create(w http.ResponseWriter, r *http.Request) {
...
 
 // to edit VA
-func (va *VAHandler) Edit(w http.ResponseWriter, r *http.Request) {
+func (va *VAHandler) Update(w http.ResponseWriter, r *http.Request) {
+
+	// set response header
+	w.Header().Set("Content-Type", "application/json")
+
+	// get va number in url
+	vaNumber := chi.URLParam(r, "va_num")
```
Comment (celynolivia) [inline api/virtual_accounts/virtual_accounts.go:300]: after receiving the va number, maybe it is better to check is the va number is valid or not, you can use the checkAccountVA function from file database_virtual_accounts.go

**Review by celynolivia — APPROVED** [review]

**Review by david1312 — APPROVED** [review]
good to go

**Pull request merged**